src/main/kotlin/com/github/pool_party/pull_party_bot/commands/PingCommandHandlers.kt

```diff
@@ -173,15 +173,25 @@ private fun Bot.handlePartyChangeRequest(isNew: Boolean, msg: Message, args: Str
         return
     }
 
-    val partyName = parsedList[0].removePrefix("@")
+    val partyName = parsedList[0].replace("@", "")
```

Comment:
You say [there](https://github.com/pool-party/pull-party-bot/pull/39/files#diff-193284190c1f6c1a40f07babddcad831R206), that `@` are not allowed and here you just skip it 🤔